components/retro-go/targets/fri3d-2024/config.h

```diff
@@ -0,0 +1,114 @@
+/* Configuration for Fri3D Camp 2024 Badge, based on the ESP32-S3.
+ *
+ * Hardware info: https://github.com/Fri3dCamp/badge_2024_hw
+ *
+ * Fri3D Camp info: https://fri3d.be/
+ *
+ */
+
+// Target definition
+#define RG_TARGET_NAME             "FRI3D-2024"
+
+// Storage
+#define RG_STORAGE_DRIVER           1                   // 0 = Host, 1 = SDSPI, 2 = SDMMC, 3 = USB, 4 = Internal Flash
```

Comment:
I've changed the format a bit, can you update it? At first glance the following would be correct:
````c
// Storage
#define RG_STORAGE_ROOT             "/sd"
#define RG_STORAGE_SDSPI_HOST       SPI2_HOST
#define RG_STORAGE_SDSPI_SPEED      SDMMC_FREQ_DEFAULT
// #define RG_STORAGE_SDMMC_HOST       SDMMC_HOST_SLOT_1
// #define RG_STORAGE_SDMMC_SPEED      SDMMC_FREQ_DEFAULT
#define RG_STORAGE_FLASH_PARTITION  "vfs"
````

Reply:
I can merge as is and fix it myself as well, if you prefer.